contracts/src/ERC20Destination.sol

```diff
@@ -75,8 +75,6 @@ contract ERC20Destination is IERC20Bridge, TeleporterTokenDestination, ERC20 {
      * @dev See {TeleportTokenDestination-_deposit}
      */
     function _deposit(uint256 amount) internal virtual override returns (uint256) {
-        // TODO: can copy logic from SafeERC20TransferFrom.safeTransferFrom directly
-        // figure out if has gas savings.
         return SafeERC20TransferFrom.safeTransferFrom(this, amount);
```

Comment:
can you add a comment explaining this for internal library functions

Reply:
IMO a comment here is not necessary, otherwise we'd want to leave similar comments at every internal library call to remain consistent.

Reply:
yup agreed, ideally I'd want to add that comment in `teleporter` where the actual library code is, but thought this was easier. Not a big pref either way.